src/content/docs/en/developers/developer-quickstart.mdx

```diff
@@ -112,6 +114,14 @@ To deploy using the Scroll Sepolia Testnet Public RPC, run:
 forge create ... --rpc-url=https://sepolia-rpc.scroll.io/ --legacy
 ```
 
+### Atlas
+
+1. Go to https://app.atlaszk.com
+2. Go to the top right and change the network to Scroll
+3. Press Deploy
+
+That's it!
```

Comment:
Please remove. Our writing style is to remain neutral and avoid language that leans toward marketing, especially in articles that lean toward "reference material".